### DISCUSSION TITLE
[KDE] How to open a new window without having to manually click the "Open new window" button?

### DISCUSSION THREAD

**Discussion by sleeplessKomodo** [FreeTubeApp/FreeTube#5361] [Q&A]
Hey, amazing application, if I haven't said so elsewhere before!

I just wanted to know, because I haven't yet seen it mentioned somewhere else, but similar to applications like Firefox or VLC, in a desktop environment like KDE, where users may have multiple virtual desktops, when a launcher or command is used (eg. `freetube`, `vlc` . . . ), a new window will be opened on the current virtual desktop. However, with freetube, if the application is currently already open on a different virtual desktop, when attempting to open a new window via a launcher or commandline, the window manager will switch focus to the already opened freetube application, _instead of opening a window_. 

Am I missing something here, or is there no way to open a new freetube window without clicking the "Open new window" button from within the application?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi there, you arent missing anything. It should open a new window but FreeTube is told to just focus the current window right now. This issue is being tracked in https://github.com/FreeTubeApp/FreeTube/issues/5153
